include/sequence.h

```diff
@@ -323,4 +325,18 @@ typedef enum NatureAmimalId {
 
 #define NATURE_IO_ENTRIES_END 0xFF
 
+// functions in sequence.c
+
```

Comment:
Do you plan to add more function declarations to this file? sequence.c seems to be the default place to look anyway, so I don't think this comment is necessary

Reply:
Yes MM has more